src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -828,9 +828,9 @@ public function iSaveMyNewConfiguration()
     /**
      * @When I set the position of :productName to :position
      */
-    public function iSetThePositionOfTo($productName, $position)
+    public function iSetThePositionOfTo(string $productName, string $position): void
```

Comment:
`int $position`?

Reply:
The idea is to allow setting a `string` as it's possible from the UI (you can type any string in the position field) and therefore we can check if validation is working